website/content/en/docs/main/getting-started/users.md

```diff
@@ -79,7 +77,7 @@ Deploy ClusterLink in a console with access to the cluster (step 3).
 
 ## Try it out
 
-Check out the [ClusterLink Tutorials][] for setting up multi-cluster connectivity
+Check out the [ClusterLink Tutorials][tutorials] for setting up multi-cluster connectivity
```

Comment:
ditto. We intentionally used `[]`. What is the rationale for changing it?

Under markdown you can have
- inline targets: `[link text](link target)`;
- named targets: `[link text][target name]` followed by `[target name]: link target` elsewhere in the file; and
- targets named by link text: `[link text][]` which uses `link text` as the `target name`